### ISSUE TITLE
page request: pacman

### ISSUE THREAD

**Issue by GingkathFox** [tldr-pages/tldr#5031]
[Docs](https://www.archlinux.org/pacman/)

**Comment by navarroaxel** [comment]
Hi @GingkathFox the `pacman` command is available here: https://github.com/tldr-pages/tldr/blob/master/pages/linux/pacman.md

Do you think we should update or change some examples in that page?

**Comment by GingkathFox** [comment]
are the pages unique to a OS? my tldr installation on macOS cant find it

**Comment by navarroaxel** [comment]
the pages in the `common/` folder are for all the OS, but you have platform specific folders like `linux/`.

I don't know which client are you using, but there is a `--platform` argument to request pages for specific platforms. https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#arguments

Maybe you need to access `linux/` pages from your macOS to use the commands in a Linux VM, right?

**Comment by GingkathFox** [comment]
kindof? i have a manjaro vm in my linux boot but whenever i try to spoof my mac address on mint it breaks the wifi, so i have to rely on macos to read the docs. Thanks!

**Comment by sbrl** [comment]
Hey there, @GingkathFox! `tldr pacman` works for me:

![sbrl@Ramblewood: ~_2020-12-18_15:16:18_001_e5128e3](https://user-images.githubusercontent.com/9929737/102630066-fbea9080-4143-11eb-860b-f514e2b0b583.png)

Depending on your client, it won't search other platforms for pages - e.g. if you're on macOS it won't show you `linux` pages. This is an issue you need to take up with the client you've installed.

As a workaround, you can do the following:

```bash
tldr -p PLATFORM_NAME PAGE_NAME
```

For example to look at the `pacman` page (which is filed under `linux`), then you'd do this:

```bash
tldr -p linux pacman
```

This should work for all clients compliant with the [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md).

**Comment by GingkathFox** [comment]
ye, i wish i could see the tldr for other platforms without the -p option

**Comment by sbrl** [comment]
Some clients do support that @GingkathFox. For example, I use [tldr-bash-client](https://github.com/pepa65/tldr-bash-client) myself.
